core/shared/src/main/scala/org/http4s/headers/Keep-Alive.scala

```diff
@@ -52,11 +52,11 @@ keep-alive-extension = token [ "=" ( token / quoted-string ) ]
       extension: List[(String, Option[String])],
   ): ParseResult[`Keep-Alive`] =
     if (timeoutSeconds.isDefined || max.isDefined || extension.nonEmpty) {
-      val reservedTokens = List("token", "max")
-      if (extension.exists(p => reservedTokens.contains(p._1))) {
+      val reservedToken = "token"
```

Comment:
Isn't that supposed to be `val reservedToken = "timeout"`? There's nothing in the RFC about "token". And it would make sense that the predefined names ("timeout", "max") cannot be used as extension names.

Reply:
I think you're very much on point. Now thinking of it, I couldn't really work out why the original code mentioned "token". Thanks!